### PR TITLE
Tweak events and supporters for charity profile app

### PR DESCRIPTION
**Pull request by benoneal**
Whole bunch of changes to accommodate the charity profile app launch requirements.

### CODE REVIEW

File: src/components/callstoaction/Button/style.scss [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  color: $ehw-grey-dark;
+}
+.Button--disabled {
+  color: $ehw-grey-dark;
+  border-color: $ehw-grey-dark;
+  background-color: transparent;
+  border-style: dashed;
+  cursor: default;
+}
+.Button--disabled:before {
```
Comment (bjbrewster) [inline src/components/callstoaction/Button/style.scss]: Don't like sass nesting?

File: src/components/events/Event/index.js [hunk 2]
```diff
@@ -119,7 +110,7 @@ module.exports = React.createClass({
           </ul>
           <a href={ props.campaignUrl } className="Event__name">{ props.name }</a>
           <p className="Event__supporter-count">{ (props.supporterCount || 0) + ' ' + t('supportersLabel') }</p>
-          <a href={ props.getStartedUrl } className="Event__join-event">{ t('joinLabel') }</a>
+          <Button kind="secondary" reverse={ true }href={ props.getStartedUrl } className="Event__join-event">{ t('joinLabel') }</Button>
```
Comment (bjbrewster) [inline src/components/events/Event/index.js]: Missing space before `href`

File: src/components/events/Event/style.scss [hunk 3]
```diff
@@ -118,16 +120,9 @@ $ehw-animation-speed: 200ms;
...
+  left: 50%;
   top: 110%;
-  width: 40%;
-  text-decoration: none;
   transition-delay: 200ms;
-  background-color: $ehw-green;
-  border-radius: 3px;
-  box-shadow: -1px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  margin: 0;
+  transform: translateX(-50%) translateY(-50%);
```
Comment (bjbrewster) [inline src/components/events/Event/style.scss]: Does this work IE8/9 ?

File: src/components/supporters/SupporterCard/index.js [hunk 4]
```diff
@@ -30,12 +30,16 @@ module.exports = React.createClass({
...
+            size={ 140 }
             userImage={ props.image }
             userName={ props.name }
             userUrl={ props.url } />
 
-          <a href={ props.url } className="SupporterCard__userName">{ props.name }</a>
+          <a href={ props.url } className="SupporterCard__userDetails">
+            <div className="SupporterCard__name">{ props.name }</div>
+            { (props.charity && !props.hideCharity) && <div className="SupporterCard__charity">For <strong>{ props.charity }</strong></div> }
+            { (props.event && !props.hideEvent) && <div className="SupporterCard__event">Through <strong>{ props.event }</strong></div> }
```
Comment (bjbrewster) [inline src/components/supporters/SupporterCard/index.js]: Can you please add `charity`, `event`, `hideCharity` and `hideEvent` to propTypes?

File: src/components/events/Event/style.scss (hunk 3 above)
Comment (benoneal) [inline src/components/events/Event/style.scss]: Pretty confident we've dropped / are very soon dropping support for those browsers.

File: src/components/callstoaction/Button/style.scss (hunk 1 above)
Comment (benoneal) [inline src/components/callstoaction/Button/style.scss]: Was converted to css from stylus. No real point spending time manually converting it to sass :P

File: src/components/supporters/Supporters/index.js [hunk 5]
```diff
@@ -101,6 +95,10 @@ module.exports = React.createClass({
         url={ page.url }
         image={ page.image.large_image_url }
         name={ page.name }
+        hideCharity={ props.hideCharityName }
+        hideEvent={ props.hideEventName }
+        charity={ page.charity_name }
```
Comment (bjbrewster) [inline src/components/supporters/Supporters/index.js]: Can we do the hide tests here? Like `{ props.hideCharityName && page.charity_name}` ?

File: src/components/events/Event/style.scss (hunk 3 above)
Comment (bjbrewster) [inline src/components/events/Event/style.scss]: Should work with IE9 at least.

**Pull request merged**